R/legend-draw.R

```diff
@@ -236,7 +236,6 @@ draw_key_smooth <- function(data, params, size) {
 #' @export
 #' @rdname draw_key
 draw_key_text <- function(data, params, size) {
-  if(is.null(data$label)) data$label <- "a"
```

Comment:
I'm a little bit worried with this line.
I fear that the following code wouldn't draw labels in the legend:

```r
ggplot(mtcars, aes(mpg, disp)) +
  geom_text(aes(
    label = rownames(mtcars), 
    colour = factor(cyl)
  ))
```

Could you confirm and amend if necessary?